src/AddIns/Uno.UI.MediaPlayer.Skia.Gtk/MediaPlayerPresenterExtension.cs

```diff
@@ -39,6 +39,9 @@ public MediaPlayerPresenterExtension(object owner)
 		_owner = presenter;
 		_player = new GtkMediaPlayer(_owner);
 		var contentView = new ContentControl();
+
+		contentView.EffectiveViewportChanged += (_, _) => _player.UpdateVideoLayout();
```

Comment:
Do we need to worry about unsubscribing from this event at any point?

Reply:
I'm assuming the lifetime of the contentView matches that of the presenter itself. In that case, shouldn't we be ok? cc. @RafaelMendesRosa